viewer/src/components/Game.vue

```diff
@@ -233,7 +233,7 @@ export default class Game extends Vue {
 
   get canPlay() {
     return (
-      !this.ended && (this.sessionPlayer === this.engine.players[this.player])
+      !this.ended && (!this.$store.state.gaiaViewer.player || this.sessionPlayer === this.engine.players[this.player])
```

Comment:
0 is a valid value for `player`, it should be tested against `null` or `undefined` (probably `undefined`) with `===`

Reply:
I just reverted my accidental change - player is `player: null as { index?: number; auth?: string } | null,` here